From e5074e5038ec7f41752133a35c0f36ef9c25d719 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Gabriel=20Iv=C4=83ncescu?= <gabrielopcode@gmail.com>
Date: Tue, 9 Jul 2019 14:19:01 +0300
Subject: [PATCH] Define AT_NO_AUTOMOUNT if needed.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Old kernel headers don't define this flag, so we do it ourselves in that case.

Wine-Bug: https://bugs.winehq.org/show_bug.cgi?id=47445
Signed-off-by: Gabriel IvÄƒncescu <gabrielopcode@gmail.com>
---
 dlls/ntdll/directory.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dlls/ntdll/directory.c b/dlls/ntdll/directory.c
index b50d41bb80..e3f805a48f 100644
--- a/dlls/ntdll/directory.c
+++ b/dlls/ntdll/directory.c
@@ -142,6 +142,10 @@ typedef struct
 # define O_DIRECTORY 0200000 /* must be directory */
 #endif
 
+#ifndef AT_NO_AUTOMOUNT
+#define AT_NO_AUTOMOUNT 0x800
+#endif
+
 #endif  /* linux */
 
 #define IS_OPTION_TRUE(ch) ((ch) == 'y' || (ch) == 'Y' || (ch) == 't' || (ch) == 'T' || (ch) == '1')
-- 
2.17.1

